contracts/artifacts/sale/IERC1155Sale.sol/IERC1155Sale.json

```diff
@@ -0,0 +1,712 @@
+{
```

Comment:
please rename this file to.. `contracts/artifacts/seq_sale/IERC1155Sale.sol/IERC1155Sale.json`